app/assets/images/govuk_publishing_components/chevron-banner/chevron-banner-focus.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg viewBox="0 0 132 109" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<defs>
+<path id="d" d="M0 0h90v109H0z"/>
+</defs>
+<g fill="#FD0" shape-rendering="geometricPrecision" stroke="#000" stroke-width="2" vector-effect="non-scaling-stroke">
+<path d="m88.611 52.664c1.0371 1.1432 1.0375 2.8869 8.02e-4 4.0305l-44.991 49.634c-0.37092 0.40919-0.33989 1.0416 0.069307 1.4125 0.18394 0.16674 0.42334 0.25909 0.6716 0.25909h36.892c0.55978 0 1.0939-0.2346 1.4727-0.6468l46.534-50.643c1.0549-1.148 1.0546-2.9127-7.09e-4 -4.0604l-46.898-51.003c-0.37873-0.41188-0.91268-0.64628-1.4722-0.64628h-36.891c-0.55228 0-1 0.44772-1 1 0 0.24841 0.092454 0.48792 0.25936 0.6719l45.352 49.992z" vector-effect="non-scaling-stroke"/>
+<mask id="c" fill="#fff">
+<use xlink:href="#d"/>
+</mask>
+<path d="m-40.997 1l0.35768 107h40.408c0.55978 0 1.0939-0.2346 1.4727-0.6468l46.534-50.643c1.0549-1.148 1.0546-2.9127-7.095e-4 -4.0604l-46.898-51.003c-0.37873-0.41188-0.91268-0.64628-1.4722-0.64628h-40.402z" mask="url(#c)" vector-effect="non-scaling-stroke"/>
+</g>
+</svg>
```

Comment:
Just tried running this through https://jakearchibald.github.io/svgomg/ and it reckons it can compress it to 75% of its current size.